packages-next/admin-ui/src/utils/useAdminMeta.tsx

```diff
@@ -93,7 +93,7 @@ export function useAdminMeta(adminMetaHash: string, fieldViews: FieldViews) {
             );
           }
         });
-        const views = fieldViews[field.viewsIndex];
+        const views = { ...fieldViews[field.viewsIndex] };
```

Comment:
I could not trace why this `views`  has Field as getter only, best way would be to make sure this allows setter, but this is quick fix.

Reply:
Can you show an example of how you defined your custom field views? I'd like to try to recreate this so we can get to the root cause of the bug.